src/python/coupled1D0D.py

```diff
@@ -124,7 +124,7 @@
     print " == >> Reading geometry from files... << == "
 
 # Read the node file
-with open('Input/Node.csv','rb') as csvfile:
+with open('input/Node.csv','rb') as csvfile:
```

Comment:
This needs to be more generic and accept the node input file as an argument to the script using this as the default location.  Same applies to other input files.